nifi-nar-bundles/nifi-standard-services/nifi-oauth2-provider-bundle/nifi-oauth2-provider-service/pom.xml

```diff
@@ -74,7 +74,7 @@
         <dependency>
             <groupId>com.squareup.okhttp3</groupId>
             <artifactId>okhttp</artifactId>
-            <version>3.14.4</version>
+            <version>${okhttp.version}</version>
```

Comment:
Any reason to use the "externalized" version for NiFi framework here but not for the other bundles? The OAuth2 bundle is not part of the framework pieces I believe.

Reply:
Thanks for the feedback @pvillard31, that's a good point, I didn't intend to use the framework version in this bundle, will make the change here to use the explicit version.